CODE_OF_CONDUCT.md

```diff
@@ -28,6 +28,9 @@ bar for ourselves.
   all beginners once, and can teach and learn through feedback and sharing.
 - **are open to feedback** from others, be it technical points, interpersonal
   tips, or language/wording suggestions.
+- **accept responsibility** by applogizing to those affected by our mistakes,
+  and we graciously accept applogies knowing that mistakes are simply part of
+  the change process.
```

Comment:
*apologizing
*apologies

"the change process." sounds odd here, how about "the process of change" or simply "the process"